extensions/wikia/UserLogin/tests/UserLoginTest.php

```diff
@@ -1,782 +1,869 @@
 <?php
```

Comment:
Ok.  This entire file was tabbed in one stop for some reason.  I fixed the tabbing but it makes this messy.  I'll note the only addition I made to this file below.